### ISSUE TITLE
Update 'class-transformer' to version '^0.5.1'

### ISSUE THREAD

**Issue by yeswell** [dmitriy-nz/nestjs-form-data#21]
I have problem similar with [Issue #20](https://github.com/dmitriy-nz/nestjs-form-data/issues/20), but with package 'class-transformer' 

Please, update 'class-transformer' to version '^0.5.1'

```
<PROJECT-DIR>$ npm install --save-exact nestjs-form-data
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: <PROJECT-NAME>
npm ERR! Found: class-transformer@0.5.1
npm ERR! node_modules/class-transformer
npm ERR!   class-transformer@"0.5.1" from the root project
npm ERR!   peerOptional class-transformer@"*" from @nestjs/common@8.0.0
npm ERR!   node_modules/@nestjs/common
npm ERR!     @nestjs/common@"8.0.0" from the root project
npm ERR!     peer @nestjs/common@"^7.0.0 || ^8.0.0" from nestjs-form-data@1.6.1
npm ERR!     node_modules/nestjs-form-data
npm ERR!       nestjs-form-data@"*" from the root project
npm ERR!     5 more (@nestjs/core, @nestjs/microservices, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer class-transformer@"^0.4.0" from nestjs-form-data@1.6.1
npm ERR! node_modules/nestjs-form-data
npm ERR!   nestjs-form-data@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See <HOME-DIR>/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     <HOME-DIR>/.npm/_logs/<DATE>-debug-0.log
```

Many thanks for your wonderful work!

**Comment by gustavoburbano** [comment]
Same here, it would be nice to have it updated.
Thanks for your work, it really helps a lot! :)

**Comment by dmitriy-nz** [comment]
Hi, thanks for your positive feedbacks. I have published version [nestjs-form-data@1.6.2](https://www.npmjs.com/package/nestjs-form-data) check if the issue is fixed.
Best Wishes
